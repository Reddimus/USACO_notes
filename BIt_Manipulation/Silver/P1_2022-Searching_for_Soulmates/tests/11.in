10
2584580044 3309429084
3574958677 4181380107
835687912 4195741841
3298860682 1982521165
2413958636 3792116170
151228585 2586624805
489758793 2585515968
3653213849 2264688676
2310472741 1654433825
1037945201 2471008197
