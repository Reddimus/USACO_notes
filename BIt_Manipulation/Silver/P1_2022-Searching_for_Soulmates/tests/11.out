83
86
77
88
83
74
87
81
85
80
